.github/labeler.yml

```diff
@@ -29,6 +29,10 @@
 - .github/ci_commit_pins/**
 - c10/core/Sym*
 - torch/fx/experimental/symbolic_shapes.py
+- test/distributed/_tensor/test_dtensor_compile.py
+- test/distributed/tensor/parallel/test_fsdp_2d_parallel.py
```

Comment:
I'm sorta curious how to only trigger `test_inductor_distributed` if we are using a universal "ciflow/inductor" label? or the label would just trigger both `test_inductor` and `test_inductor_distributed`?

Reply:
One can introduce new label (see how it's done for `ciflow/mps`), but I wonder how isolated you want to be